SD-K9/src/app/app-routing.module.ts

```diff
@@ -2,24 +2,28 @@ import { NgModule } from '@angular/core';
 import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
-  { path: '', redirectTo: 'home', pathMatch: 'full' },
+  { path: '', redirectTo: 'home-temp', pathMatch: 'full' },
```

Comment:
I didn't want to modify the home.page to test the new design. It was simpler to create a new page: home-temp.page. There are also a lot of code in the home.page that can be converted to a component. Since home.page can use some cleaning up, I didnt add my dummy code on top of it to make it messier.

Reply:
Okay, we will create an issue to clean up home.page in that case